clusterman/cli/util.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+import signal
+
+import colorlog
+
+logger = colorlog.getLogger(__name__)
+
+
+def timeout_wrapper(main):
+    def wrapper(args: argparse.Namespace):
+
+        # After 10s, prints a warning message if the command is running from the wrong place
+        def timeout_handler(signum, frame):
+            logger.warning(
+                'This command is taking a long time to run; are you running from the right account?\n'
+                'HINT: try ssh\'ing to adhoc-prod or another box in the right account.'
+            )
+
+        signal.signal(signal.SIGALRM, timeout_handler)
+        signal.alarm(10)
```

Comment:
For posterity, I think it's better if you made the timeout duration configurable. You could also go the extra step of making the signal configurable, but I think `SIGALRM` is less likely to change.

Reply:
I agree that it should not be a magic number but I don't think exposing it as a config option makes sense.  I'll make that change.